schema_utils_test.py

```diff
@@ -113,6 +108,27 @@
     },
 }
 
+# Schemas for the UI config for the schemas of type equal to 'custom'. Maps
+# schemas obj_type to the UI config schemas.
+OBJ_TYPE_TO_UI_CONFIG_SPECS = {
```

Comment:
I still don't get why you're doing all this, and I also don't think it's correct. Why can't you add the appropriate ui config options to the (inner) schema of the relevant objects?

Reply:
That's not what I suggested at all -- and those seem like rather strange primitive types. Could you take a look again at the question I posed?

Reply:
Hey Sean, sorry for the confusion. When we talked on Friday, you asked me why 'SubtitledHtml' and 'SubtitledUnicode' had to be {type: 'custom'}, and I thought we agreed that they could be their own type, which is what I changed.

The question above was relevant to including Ui config specs in custom typed objects, but now that is not relevant as 'SubtitledHtml' and 'SubtitledUnicode' are their own types now. Does this make sense?

Reply:
Just to double-check ... I think we talked about this, is everything clearer now?

Reply:
Yes- thanks

Reply:
Updated all {type: SubtitledUnicode/SubtitledHtml} locations to a dict schema with the appropriate fields. I had to add helper functions to identify if a given schema is a subtitledhtml/subtitledunicode schema, which complicated things slightly.